headers/functions/overlay29.h

```diff
@@ -392,12 +412,24 @@ bool IsInSpawnList(undefined* spawn_list, enum monster_id monster_id);
 int ChangeShayminForme(struct entity* entity, int forme);
 void ApplyItemEffect(undefined4 param_1, undefined4 param_2, undefined4 param_3,
                      struct entity* attacker, struct entity* defender, struct item* thrown_item);
+void ApplyCheriBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyPechaBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyRawstBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyHungerSeedEffect(struct entity* attacker, struct entity* defender);
+void ApplyVileSeedEffect(struct entity* attacker, struct entity* defender);
 void ViolentSeedBoost(struct entity* attacker, struct entity* defender);
```

Comment:
Let's change this to `user` and `target` with the rest of them.

Reply:
I believe I made most of them attacker and defender because that's what they're called in ApplyItemEffect. Will change them to `user` and `target`. Would you like me to also change the function for ViolentSeedBoost to user target and/or change its name to ApplyViolentSeedBoost to match the other functions?

Reply:
Yeah let's change all the item effect stuff to use "user" and "target" (in both the symbol descriptions and the function headers; what we say in one should match the other). The attacker/defender terminology comes from the damage calcs and move effects, but I don't think it really makes sense for items since there are so many items that don't fit that mold.

> Would you like me to also change the function for ViolentSeedBoost to user target and/or change its name to ApplyViolentSeedBoost to match the other functions?

Yeah that sounds good to me. The item effects should be consistent, like how the move effects are.